qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -656,6 +689,9 @@
     </item>
     <item>
      <widget class="QToolButton" name="channelRed">
+      <property name="toolTip">
+       <string>Show Red</string>
```

Comment:
I'd rather have the tooltip correct now, in case whoever implements the right click forgets to update it. This is no different to having buttons that aren't implemented or other functionality missing but 'presented' elsewhere in the UI.